websites/N/Nakama Mangas/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "author": {
+    "name": "Nanashin",
+    "id": "330030557457022986"
+  },
+  "service": "Nakama Mangás",
+  "description": {
+    "en": "One brazilian scanlator with a quality website.",
+    "pt": "Uma scanlator brasileira com um ótimo conteudo e site.",
+    "pt_BR": "Uma scanlator brasileira com um ótimo conteudo e site."
+  },
+  "url": "nakamamangas.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/QDPO5Yr.gif",
+  "thumbnail": "https://imgur.com/a/rVO1bJI",

```

Comment:
Please use the direct link of the image

Reply:
it's the direct link: 
![image](https://user-images.githubusercontent.com/63855357/82256239-5a405e00-992c-11ea-95e1-7634b5ec9567.png)

Reply:
he's referring to the thumbnail

Reply:
and if the logo is not an animation, can you convert it to png/jpg